devices/models.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+
+from users.models import User
+
+
+class Building(models.Model):
+    name = models.CharField(max_length=100, null=True)
+    user = models.ForeignKey(
+        User, related_name="user_buildings", null=True, on_delete=models.SET_NULL
+    )
+
+    def __str__(self):
+        return f"Building: {str(self.id)} | name: {self.name}"
+
+
+class Room(models.Model):
+    name = models.CharField(max_length=100, null=False)
```

Comment:
I think it would be better to store max_length as some Constant structure, because it occures multiple times.

Reply:
django convention